CSS/main.css

```diff
@@ -0,0 +1,161 @@
+.body {
+  font-family: "Lato", sans-serif;
+}
+.page_header {
+  background: linear-gradient(
+    90deg,
+    rgba(213, 32, 71, 0.8) 0%,
+    rgba(238, 61, 27, 0.8) 100%
+  );
+  color: #ffffff;
+}
+.logo-img {
+  line-height: 29px;
+}
+.main-nav {
+  color: #ffffff;
```

Comment:
maybe #fff. Check all the project

Reply:
But in our hometask you wrote we have to use only "hex" and "rgba" image format. If i write #fff, it will be rgb format.
I checked it out. I can show you it in prtsc. Look on your telegram.